DEVELOPMENT.md

```diff
@@ -297,6 +298,24 @@ machines.
   _always_ include the reason the note is needed. Remember, more information is better!
 
 
+### Usage Docs
+
+Usage docs should be added to all components and directives. These docs are created by adding a markdown file named to match the component
+or directive:
+
+```
+// component:
+foo.component.ts
+// usage docs:
+foo.component.md
+```
+
+When the documentation published, this usage doc will be automatically consumed by the API docs generation.
+
+Whenever a usage document is edited, `yarn run docs:toc` should be run. This will automatically update the Table of Contents at the top of
```

Comment:
This is the wording that confused me.

To edit a doc, add headlines and appropriate content, then 'yarn run docs:toc` to update the table of contents.